diff --git a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/ChainConfig.js b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/ChainConfig.js
index 007d740..139c489 100644
--- a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/ChainConfig.js
+++ b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/ChainConfig.js
@@ -10,6 +10,13 @@ exports.chainConfig = {
             browserURL: "https://etherscan.io",
         },
     },
+    celo: {
+        chainId: 42220,
+        urls: {
+            apiURL: "https://api.celoscan.xyz/api",
+            browserURL: "https://celoscan.xyz",
+        },
+    },
     harmony: {
         chainId: 1666600000,
         urls: {
diff --git a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/types.d.ts b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/types.d.ts
index 0a9efc3..88bdaac 100644
--- a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/types.d.ts
+++ b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/types.d.ts
@@ -1,4 +1,4 @@
-declare type Chain = "mainnet" | "ropsten" | "rinkeby" | "goerli" | "kovan" | "bsc" | "bscTestnet" | "heco" | "hecoTestnet" | "opera" | "ftmTestnet" | "optimisticEthereum" | "optimisticKovan" | "polygon" | "polygonMumbai" | "arbitrumOne" | "arbitrumTestnet" | "avalanche" | "avalancheFujiTestnet" | "moonbeam" | "moonriver" | "moonbaseAlpha" | "harmony" | "harmonyTest" | "xdai" | "sokol" | "aurora" | "auroraTestnet";
+declare type Chain = "celo" | "mainnet" | "ropsten" | "rinkeby" | "goerli" | "kovan" | "bsc" | "bscTestnet" | "heco" | "hecoTestnet" | "opera" | "ftmTestnet" | "optimisticEthereum" | "optimisticKovan" | "polygon" | "polygonMumbai" | "arbitrumOne" | "arbitrumTestnet" | "avalanche" | "avalancheFujiTestnet" | "moonbeam" | "moonriver" | "moonbaseAlpha" | "harmony" | "harmonyTest" | "xdai" | "sokol" | "aurora" | "auroraTestnet";
 export declare type ChainConfig = {
     [Network in Chain]: EtherscanChainConfig;
 };
diff --git a/node_modules/@nomiclabs/hardhat-etherscan/src/types.ts b/node_modules/@nomiclabs/hardhat-etherscan/src/types.ts
index b88e8fa..4c658b2 100644
--- a/node_modules/@nomiclabs/hardhat-etherscan/src/types.ts
+++ b/node_modules/@nomiclabs/hardhat-etherscan/src/types.ts
@@ -36,7 +36,7 @@ type Chain =
   | "sokol"
   // aurora
   | "aurora"
-  | "auroraTestnet";
+  | "auroraTestnet" | "celo";
 
 export type ChainConfig = {
   [Network in Chain]: EtherscanChainConfig;
